test/integration/edition_edit_test.rb

```diff
@@ -161,6 +196,73 @@ class EditionEditTest < IntegrationTest
     end
   end
 
+  context "Update important note page" do
+    should "render the 'Update important note' page" do
+      visit_draft_edition
+      click_link("History and notes")
+      click_link("Update important note")
```

Comment:
We could go straight to the page
```suggestion
      create_draft_edition
      visit history_update_important_note_edition_path(@draft_edition)
```